packages/rabbitmq/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@cardano-sdk/rabbitmq",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.2.0",
```
Package versions are synced across the SDK